modules/cnf-verifying-performance-setting.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="verifying-performance-configuration_{context}"]
+= Verifying performance configuration
+
+.Procedure
+
+Follow this procedure to verify the performance settings applied to your cluster.
+
+. Start a debugging session on one of your nodes that is targeted by the performance profile by running this command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Set `/host` as the root directory within the debug shell by running this command:
++
+[source,terminal]
+----
+sh-4.4# chroot /host
+----
+
+. Display the kernel command line parameters without any performance profile applied by running the following command:
++
+[source,terminal]
+----
+$ cat /proc/cmdline
+----
++
+.Example output
++
+[source,terminal]
+----
+systemd.unified_cgroup_hierarchy=1 cgroup_no_v1=all psi=1
+----
++
+.Example 1 
+
+With the following low latency performance profile applied to a single-node OpenShift cluster.
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: 3-15
+    reserved: 0-2
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    highPowerConsumption: false
+    perPodPowerManagement: false
+    realTime: true
+----
++
+The kernel parameters that deliver both energy savings and low-latency are as follows: 
++
+[source,terminal]
+----
+skew_tick=1 tsc=reliable rcupdate.rcu_normal_after_boot=1 nohz=on rcu_nocbs=3-15 tuned.non_isolcpus=00000007 systemd.cpu_affinity=0,1,2 intel_iommu=on iommu=pt isolcpus=managed_irq,3-15 nohz_full=3-15 tsc=reliable nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11 intel_pstate=disable systemd.unified_cgroup_hierarchy=0 systemd.legacy_systemd_cgroup_controller=1
+----
++
+.Example 2 
++
+With the following lowest latency performance profile applied to a single-node OpenShift cluster.
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
+    isolated: 3-15
+    reserved: 0-2
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+  workloadHints:
+    highPowerConsumption: true
+    perPodPowerManagement: false
+    realTime: true
```

Comment:
I don't think we recommend the use of highPowerConsumption for any of our customer use cases - do we?

Reply:
we do not